components/resources/quizzes/shared/SkillSelections.tsx

```diff
@@ -18,15 +20,22 @@ const SkillSelections: React.FC<SkillSelectionsProps> = ({
   progress,
   title,
   body,
+  maxSelections,
 }) => {
   const [selected, setSelected] = useState<string[]>([]);
+  useEffect(() => {
```

Comment:
useEffect to clear the selections. @jimip85 you might find this helpful for your list datastrcture that captures all the users selections.